controllers/useraccount/console_usersettings_test.go

```diff
@@ -0,0 +1,144 @@
+package useraccount
+
+import (
+	"context"
+	"fmt"
+	. "github.com/codeready-toolchain/member-operator/test"
+	"github.com/codeready-toolchain/toolchain-common/pkg/test"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+	"testing"
+)
+
+func TestDeleteConsoleSettingObjects(t *testing.T) {
+	t.Run("Object found by name and deleted", func(t *testing.T) {
+		// given
+		ctx := context.Background()
+		cm := &corev1.ConfigMap{
+			TypeMeta: metav1.TypeMeta{Kind: "ConfigMap"},
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "user-settings-johnsmith",
+				Namespace: UserSettingNS,
+			},
+		}
+		cl := fake.NewClientBuilder().WithObjects(cm).Build()
+
+		// when
+		err := deleteResource(ctx, cl, "johnsmith", cm)
```

Comment:
Don't pass the same `cm` here. Use `&corev1.ConfigMap{}` here instead. For proper testing.

Reply:
fixed in [808948b](https://github.com/codeready-toolchain/member-operator/pull/568/commits/808948b9f9d8874182e4719eb287fe97f441bab5)